pegasus/routes/v2_section_routes.rb

```diff
@@ -13,6 +14,14 @@
   redirect "/v2/sections/#{section_id}"#, 201 #BUGBUG: JQuery is barfing on the 201
 end
 
+# Get the set of sections that the current user is enrolled in.
+get '/v2/sections/membership' do JSON.pretty_generate(:duid => dashboard_user_id)
```

Comment:
:sweat_smile: oh - I see what you mean.  This was a dumb copy-paste that I forgot to remove.